wgpu/src/util/belt.rs

```diff
@@ -197,3 +224,48 @@ impl fmt::Debug for StagingBelt {
             .finish_non_exhaustive()
     }
 }
+
+struct Chunk {
+    buffer: Arc<Buffer>,
```

Comment:
Does this still need to be `Arc`?

Reply:
No, it doesn't. There also doesn't need to be a `size` field any more. Removed those things.